atum/src/test/scala/za/co/absa/atum/AtumImplicitsSpec.scala

```diff
@@ -50,17 +50,47 @@ class AtumImplicitsSpec extends AnyFlatSpec with SparkTestBase with Matchers {
     val df = spark.read.json(Seq(inputData).toDS)
 
     // act
-    df.setAdditionalInfo(("additionalKey1", "additionalValue1"))
+    df.setAdditionalInfo("additionalKey1", "additionalValue1")
+    df.setAdditionalInfo("additionalKey1", "anotherAdditionalValue")
```

Comment:
maybe a quick comment like `//replaceIfExists is false by default, so this value will not be applied`

or even the not-used value can signify it:
```suggestion
    df.setAdditionalInfo("additionalKey1", "theValueWillNotBeOverwrittenByThis") // replaceIfExists is false by default
```